### PR TITLE
working with fileSummary v1

### PR DESCRIPTION
**Pull request by zarsky-broad**
Co-Authored-By: Matt Bemis <matthewbemis@users.noreply.github.com>

- [ ] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: If you changed a URL that is used elsewhere (e.g. in an email), comment about where it is used and ensure the dependent code is updated.
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [ ] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [ ] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [ ] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [ ] **Submitter**: If you're adding new automated UI tests, review the test plan with QA
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Product Owner** sign off
- [ ] **Submitter**: Verify all tests go green, including CI tests and automated UI tests.
- [ ] **Submitter**: Squash commits and merge to develop. If adding test code, merge application code and test code at the same time.
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

**Comment by MatthewBemis** [comment]
jenkins retest with config changes merged to dev

File: src/cljs/main/broadfcui/common/gcs_file_preview.cljs [hunk 1]
```diff
@@ -12,226 +12,197 @@
...
+                            [:div {:style {:marginTop "1em" :marginBottom "1em"}}
+                             [:div {} "Downloading large files through the browser may not be successful. Instead use this gsutil"]
+                             [:div {:style {:marginBottom ".5em"}} "command replacing [DESTINATION] with the local file path you wish to download to."]
+                             (style/create-code-sample
+                              (str "gsutil cp gs://" bucket-name "/" object " [DESTINATION]"))
+                             [:div {:style {:marginTop "1em"}} "For more information on the gsutil tool click "
+                              (links/create-external {:href "https://cloud.google.com/storage/docs/gsutil"} "here")]]))
+                 (when-not data-empty
+                   (labeled "Estimated download fee"
+                            (common/format-price (* size 1.1175870895385742e-10)) ; not actually a magic number, calculated from google...
```
Comment (MattPutnam) [inline src/cljs/main/broadfcui/common/gcs_file_preview.cljs]: Well, it's still a magic number. An explanation would be nice.

File: src/cljs/main/broadfcui/common/gcs_file_preview.cljs [hunk 2]
```diff
@@ -12,226 +12,197 @@
...
+                            [:span {:style {:marginLeft "1em"}}
+                             [:span {:style {:fontStyle "italic" :color (:text-light style/colors)}}
+                              " (non-US destinations may be higher)"]]))
+                 (when (or timeCreated updated md5Hash)
+                   (if show-details?
+                     [:div {}
+                      (when timeCreated (labeled "Created" (common/format-date timeCreated)))
```
Comment (MattPutnam) [inline src/cljs/main/broadfcui/common/gcs_file_preview.cljs]: I sorta like
```clojure
(some->> timeCreated common/format-date (labeled "Created"))
```
but maybe that's just me.

File: src/cljs/main/broadfcui/utils/ajax.cljs [hunk 3]
```diff
@@ -111,14 +111,12 @@
...
-            :method "POST"
-            :data data
-            :on-done (fn [{:keys [status-code status-text] :as m}]
-                       (on-done m))))))
+(defn call-martha [uri arg-map]
+  (call (assoc arg-map
+          :method "POST"
+          :url (config/martha-file-summary-url)
+          :headers (merge (@get-bearer-token-header) content-type=json)
+          :data (str "{\"uri\": \"" uri "\"}"))))
```
Comment (MattPutnam) [inline src/cljs/main/broadfcui/utils/ajax.cljs]: `(utils/->json-string {:uri uri})`

**Review by MattPutnam — APPROVED** [review]

**Comment by aweng98** [comment]
Just asking, I have seen two `PreviewSpec` tests ignored in this PR failed in several Jenkins PR jobs over a period of several weeks. Is failure in someway related to this?
e.g. https://fc-jenkins.dsp-techops.broadinstitute.org/job/swatomation-pipeline/8709/

**Comment by MatthewBemis** [comment]
@aweng98 this changes how they work in a way that can't currently tested (because bond and martha don't yet work in fiab). when they are re-enabled in the near future, they will probably be more reliable

**Pull request merged**